new-integration-tests/src/test/resources/exporter/norestport.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+query_sync:
+  url: http://coordinator:8999/
```

Comment:
Hope the hard-coded port will not block the parallel execution in future

Reply:
no it is internal , running in same ns as domain